actionlint/README.md

```diff
@@ -0,0 +1,100 @@
+# actionlint
+
+This composite GitHub Action (GHA) is supposed to be used to lint GitHub action workflows.
+
+It's basically a wrap around the `actionlint` tool, with the goal to provide a consistent way
```

Comment:
non-blocking:
Maybe call it "an abstraction of" (or "a wrapper of") rather than "a wrap around the".